ZTIA-507: Update docs to match new Gateway selectors for Access infra… (#19950)
cloudflare/cloudflare-docs: src/content/partials/cloudflare-one/gateway/selectors/all-access-app-targets.mdx

---
{}
---

> All targets secured by an [Access infrastructure application](/cloudflare-one/applications/non-http/infrastructure-apps/#modify-order-of-precedence-in-gateway).

Reviewer: ```suggestion
All [targets](/cloudflare-one/applications/non-http/infrastructure-apps/#1-add-a-target) secured by an [Access infrastructure application](/cloudflare-one/applications/non-http/infrastructure-apps/).
```

> All destination IPs and hostnames associated with an [Access Infrastructure application](/cloudflare-one/applications/non-http/infrastructure-apps/#modify-order-of-precedence-in-gateway).

Reviewer: @Michael9127 since target hostnames aren't used in DNS resolution, would this selector only apply to IPs?

> All destination IPs and hostnames associated with an [Access Infrastructure application](/cloudflare-one/applications/non-http/infrastructure-apps/#modify-order-of-precedence-in-gateway).

Reviewer: Does the selector apply to all targets, or only the targets that are behind an Access app?

> All destination IPs and hostnames associated with an [Access Infrastructure application](/cloudflare-one/applications/non-http/infrastructure-apps/#modify-order-of-precedence-in-gateway).

Reviewer: ```suggestion
All targets secured by an [Access Infrastructure application](/cloudflare-one/applications/non-http/infrastructure-apps/).
```
